isisd/isis_vty.c

```diff
@@ -927,6 +927,21 @@ DEFUN (no_isis_hello_padding,
 	return CMD_SUCCESS;
 }
 
+DEFUN (isis_threeway_adj,
+       isis_threeway_adj_cmd,
+       "[no] isis three-way-handshake",
+       NO_STR
+       "IS-IS commands\n"
+       "Enable three-way handshake\n")
+{
```

Comment:
Perhaps indicate that the same command could Enable and Disable 3 way handshake e.g. 
`"Enable/Disable three-way handshake\n")`

Reply:
I agree with @odd22